v2/http-middleware/security.go

```
@@ -54,6 +56,12 @@ func Configure(conf Config) {
 	if conf.UseUserIDCache {
 		userIDCache = map[string]string{}
 	}
+
+	client = rest.NewClient(rest.WithOpenCensusTracing())
```

There is a rest.WithBaseURL option to avoid the concatenation below.

---

`auth.GetBaseURL()` can throw an error if it hasn't been configured before calling it. Should I move creation of the default client to `getUserIDByToken`, or `panic` on an error?

---

Since you configure it on L52 I feel it should be safe to either ignore the error or panic.

---

I think I prefer leaving the panic to trusting that `auth.GetBaseURL()` won't fail because of the configure.